tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Operand.java

```diff
@@ -76,4 +77,18 @@
   default T data() {
     return asOutput().tensor().data();
   }
+
+  /**
+   * Returns the build-time shape of this operand.  May be only partially known.
+   */
+  default Shape shape(){
```

Comment:
I guess we want to mark the same methods in [`Output`](https://github.com/tensorflow/java/blob/91400f4e0f26f4324a8843599810baf346583395/tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Output.java#L40) as `@Override` now